package.json

```diff
@@ -38,6 +38,7 @@
     "clsx": "^1.1.1",
     "core-js": "^3.6.5",
     "document-register-element": "1.13.1",
+    "lodash": "^4.17.21",
```

Comment:
could we switch to `lodash-es`, that will support tree-shaking

Reply:
I think we only need to use lodash in devDependencies for testing. So it will not affect the size of our package. We can still use lodash-es import in our application.

Reply:
please review it whenever you get a chance.
👉 [test(browser): Avoid bringing in all lodash libraries](https://github.com/ZouYouShun/Procyonidae-UI-Verify-Tool/pull/5/commits/9929ec3372a604d89bae62dbe37882fab86f46ea)